docs/edge-authorizers/security-and-management.mdx

```diff
@@ -30,33 +30,33 @@ Once an edge authorizer is connected to the Aserto control plane:
 
 ## Configuration
 
-To use an edge authorizer, you'll first need to create an edge authorizer connection. Head to the Connections tab, and
+To use an Edge Authorizer, you'll first need to create an Edge Authorizer connection. Head to the Connections tab, and
 create a new connection by clicking the "Add connection" button. Select the "Edge Authorizer" provider and provide a
```

Comment:
"Select the "Edge Authorizer" and provide a..."
There is a "provider" after "Edge Authorizer" that doesn't seem necessary